radio/src/gui/128x64/model_logical_switches.cpp

```diff
@@ -445,7 +445,12 @@ void menuModelLogicalSwitches(event_t event)
           CHECK_INCDEC_MODELVAR_ZERO(event, cs->func, LS_FUNC_MAX);
           uint8_t new_cstate = lswFamily(cs->func);
           if (cstate != new_cstate) {
-            cs->v1 = cs->v2 = (new_cstate==LS_FAMILY_TIMER ? -119/*1.0*/ : 0);
+            unsigned int save_func = cs->func;
+            memset(cs, 0, sizeof(LogicalSwitchData));
```

Comment:
I would prefer this memclear is called only on ARM and CPUM2560 platforms, for 9X / 9XR I would prefer we cast to uint16_t the structure (or add a new union type) and apply a mask. I will give it a try